modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/search/test/CalendarBookingIndexerLocalizedContentTest.java

```diff
@@ -94,12 +96,12 @@ public void testJapaneseTitle() throws Exception {
 		String prefix1 = "新";
 		String prefix2 = "作";
 
-		Stream.of(
-			word1, word2, prefix1, prefix2
-		).forEach(
-			keywords -> assertFieldValues(
-				"title", LocaleUtil.JAPAN, titleMap, keywords)
-		);
+		List<String> keywords = new ArrayList<>(
```

Comment:
@jeffwu0724 If you just need to iterate, you don't need to wrap with an `ArrayList`.

Reply:
@jeffwu0724 Also, no need to have a separate variable. Just do something like

```
for (string keyword : Arrays.asList(word1, word2, prefix1, prefix2)) {
```

On the other hand, you can further inline variables like `word1`, because none of them are used twice.